app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -118,10 +118,16 @@ def experience_to_hash(experience)
         organisation_name: experience.organisation,
         working_with_children: experience.working_with_children,
         commitment: experience.commitment,
-        description: experience.details,
+        description: experience_description(experience),
       }
     end
 
+    def experience_description(experience)
+      return experience.details if experience.working_pattern.blank?
+
+      "#{experience.details} - #{experience.working_pattern}"
```

Comment:
I think the working pattern should come first, and that our concatenation of the two should use line breaks and labels with colons. Appending with a dash at the end will mean this information merges into the details, and could read poorly depending on what's input.

eg:
```
"Working pattern: #{experience.working_pattern}\n\nDescription: #{experience.details}"
```

Reply:
I think it reads much better than the dash, thanks for you suggestion, updated! 👍

Reply:
Looks good to me @fofr!

Reply:
@duncanjbrown are you happy with the rest of the code? I just want to make sure someone had a second look before I merge.